docs/learn/learn-guides-DOT-KSM-bridge.md

```diff
@@ -0,0 +1,110 @@
+tracked [here](https://forum.polkadot.network/t/polkadot-kusama-bridge/2971/1).
+
+## Transfer DOT to Kusama Asset Hub
+
+This tutorial shows how to transfer DOT on Polkadot Asset Hub to Kusama Asset Hub. The first step is
+to ensure that your account on Polkadot Asset Hub has enough DOT to cover the XCM transfer fee and
+the bridge fee (which is around 2 DOT). The next step is to craft an XCM message to be sent from
+Polkadot Asset Hub.
+
+[BagPipes (formerly called xcmsend)](https://xcmsend.com/#/builder) is an opensource application
+that lets you create workflows in a drag and drop style interface in order to build execution flows
+of cross chain assets transfers using XCM. Check
+[Bagpipes docs](https://xcmsend.github.io/workflows/dotksm.html) for more information on how to
+create workflows for crafting XCM transfers. The snapshot below shows a workflow on BagPipes that is
+designed to send 3 DOT from an account Polkadot Asset Hub to Kusama Asset Hub.
+
+![BagPipes Snapshot DOT Transfer](../assets/bridge-hub/PAH-to-KAH-DOT-transfer.png)
+
+This workflow crafts an XCM transfer as shown below.
+
+```
+{
+  "isSigned": false,
+  "method": {
+    "args": {
+      "dest": {
+        "V3": {
+          "parents": "2",
+          "interior": {
+            "X2": [
+              {
+                "GlobalConsensus": "Kusama"
+              },
+              {
+                "Parachain": "1,000"
+              }
+            ]
+          }
+        }
+      },
+      "beneficiary": {
+        "V3": {
+          "parents": "0",
+          "interior": {
+            "X1": {
+              "AccountId32": {
+                "network": null,
+                "id": "0x9e4e7009937c56d267338762a60ed004293afd40e7c2081847c12cb63c76a818"
+              }
+            }
+          }
+        }
+      },
+      "assets": {
+        "V3": [
+          {
+            "id": {
+              "Concrete": {
+                "parents": "1",
+                "interior": "Here"
+              }
+            },
+            "fun": {
+              "Fungible": "30,000,000,000"
+            }
+          }
+        ]
+      },
+      "fee_asset_item": "0",
+      "weight_limit": "Unlimited"
+    },
+    "method": "limitedReserveTransferAssets",
+    "section": "polkadotXcm"
+  }
+}
+```
+
+Once this [extrinsic](https://assethub-polkadot.subscan.io/extrinsic/6028374-2) is signed and
+submitted, it is broadcast to Polkadot Asset Hub nodes. As this is a reserve asset transfer, the DOT
+is transferred to a dedicated account on Polkadot Asset Hub and
+[the wrapped DOT is issued](https://assethub-kusama.subscan.io/extrinsic/6758392-0?event=6758392-1)
+as a foreign asset and deposited onto the destination account on Kusama Asset Hub. The foreign asset
+balances of any account on Kusama Asset Hub can be queried on-chain through the
+`foreignAssets`pallet as shown below.
+
+![Wrapped DOT Balance](../assets/bridge-hub/KAH-DOT-Balance.png)
+
+## Transfer KSM to Polkadot Asset Hub
```

Comment:
This section is incomplete

Reply:
Yeah, there is a bug in BagPipes that's messing with Kusama AssetHub's assets dropdown. Waiting for their response on the forum post. Worst case, I'll use PJS extrinsic from the tutorial you posted and push this PR today.

<img width="406" alt="image" src="https://github.com/w3f/polkadot-wiki/assets/86818441/bfa1c3b3-d559-417d-9e9b-d57d70ca37d7">